.github/workflows/gradle-data-capturing-samples-verification.yml

```diff
@@ -14,8 +14,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - name: 'GE Gradle plugin version'
-            sample-file: 'capture-ge-plugin-version/gradle-ge-plugin-version.gradle'
+          - name: 'DV Gradle plugin version'
+            sample-file: 'capture-dv-plugin-version/gradle-dv-plugin-version.gradle'
```

Comment:
I think we should avoid using `dv` in public facing repositories. 

There are a few occurrences of `dv` in this changeset.

Reply:
agreed, thanks for pointing this out.